src/win/tty.c

```diff
@@ -1751,7 +1703,9 @@ static int uv_tty_write_bufs(uv_tty_t* handle,
       }
 
       /* Parse vt100/ansi escape codes */
-      if (ansi_parser_state == ANSI_NORMAL) {
+      if (uv__vterm_state == UV_SUPPORTED) {
+        /* pass through escape codes if conhost supports them */
```

Comment:
Can you title-case and punctuate the comment?  Thanks.

Reply:
```suggestion
        /* Pass through escape codes if conhost supports them. */
```